src/routes/(app)/events/_components/event-hexagon.stories.ts

```diff
@@ -31,7 +31,7 @@ export const SingleDateHexagon = {
       description: 'Muito fixe',
       teamMemberIds: [],
       registerUrl: 'https://www.google.com/intl/pt-PT/forms/about/',
-      dateInterval: { startDate: new Date('2023-12-10') },
+      dateInterval: { startDate: new Date('2023-12-10'), endDate: new Date('2024-01-01') },
```

Comment:
```suggestion
      dateInterval: { startDate: new Date('2023-12-10'), endDate: new Date('2023-12-10') },
```
This is supposed to be the single date version of the hexagon, aka start and end in the same day! Perhaps we can add hours here too

Reply:
I'll try adding the hours

Reply:
Ok, I have added it and tested in storybook. It's looking great.